src/meili/map_matcher.cc

```diff
@@ -167,8 +190,9 @@ std::vector<MatchResult> InterpolateMeasurements(const MapMatcher& mapmatcher,
     return results;
   }
 
-  std::vector<EdgeSegment> route = MergeRoute(mapmatcher.state_container().state(stateid),
-                                              mapmatcher.state_container().state(next_stateid));
+  std::vector<EdgeSegment> route;
+  MergeRoute(mapmatcher.state_container().state(stateid),
+             mapmatcher.state_container().state(next_stateid), route);
```

Comment:
just rearranged the arguments and got rid of a redundant version of the function